base/somebroadcast.jl

```diff
@@ -0,0 +1,19 @@
+#Methods some Some can broadcast
+getindex(s::Some) = s.value
+getindex(s::Some, ::CartesianIndex{0}) = s.value
+
+iterate(s::Some) = (s.value, nothing)
+iterate( ::Some, s) = nothing
+
+ndims(::Some) = 0
+ndims(::Type{<:Some}) = 0
+
+length(::Some) = 1
+size(::Some) = ()
+axes(::Some) = ()
+
+IteratorSize(::Type{<:Some}) = HasShape{0}()
+Broadcast.broadcastable(s::Some) = s
+
+eltype(::Some{T})       where {T} = T
```

Comment:
This one isn't needed.

Reply:
One would expect it to be defined even if it’s not used by Base / stdlibs though, right?

I use `eltype(v)` where `v` is a vector a fair amount rather than `eltype(typeof(v))`

Reply:
There's the default fallback

https://github.com/JuliaLang/julia/blob/52c9e3df1301ce1ffc5da62626b8d4aadfc22089/base/array.jl#L125

Reply:
Oh I see, my bad!